config/initializers/config.rb

```diff
@@ -13,4 +13,8 @@
   # Parse numeric values as integers instead of strings.
   #
   # config.env_parse_values = false
+  ActionView::Base.field_error_proc = Proc.new do |html_tag, instance|
+    html_tag.html_safe
```

Comment:
just remove the `.html_safe` as discussed

Reply:
also should have a go at running through negative/unhappy path manual tests as current feature tests do not cover errors to any great degree.

Reply:
Could you leave a `# NOTE:` in this change indicating why this is being made (basically that we don't want to rely on Rails form fields error display, but use the GDS one instead that is managed in ...[point the classe(s) where it is]).

Also, FYI, [here it is](http://guides.rubyonrails.org/configuring.html#configuring-action-view) the default implementation for Rails field error.